rnndb/state_3d.xml

```diff
@@ -511,10 +525,28 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
         <!-- The following is used with unified uniforms (SH_UNIFORM_*) -->
         <reg32 offset="0x01024" name="UNIFORM_BASE" brief="Offset applied to PS uniform loads (granularity is 4-vectors)"/> <!-- -->
         <reg32 offset="0x01028" name="INST_ADDR" type="VIVM" brief="Memory address of shader instructions, must be aligned to 0x100"/> <!-- ICACHE -->
-        <reg32 offset="0x0102C" name="UNK0102C"/> <!-- numRT == 8 -->
+        <reg32 offset="0x0102C" name="CONTROL2">
+            <bitfield pos="7" name="SATURATE_RT4"/>
+            <bitfield pos="15" name="SATURATE_RT5"/>
+            <bitfield pos="23" name="SATURATE_RT6"/>
+            <bitfield pos="31" name="SATURATE_RT7"/>
+        </reg32>
         <reg32 offset="0x01030" name="CONTROL_EXT">
-            <!-- Not used for multiple render targets -->
-            <bitfield high="1" low="0" name="COLOR_OUTPUT_COUNT" brief="Number of outputs minus one"/>
+            <bitfield high="2" low="0" name="OUTPUT_MODE" brief="Number of outputs minus one">
+                <value value="0" name="NORMAL"/>
+                <!-- these values seem to bypass blend, which only works with 16-bit and no integers
+                     -with value 0, only 16 incorrect LSBs are written in the output channels
+                     -not sure what is actually different between these values, since PE format
+                      should contain this information??
+                -->
+                <value value="1" name="A2B10G10R10UI"/>
+                <value value="2" name="UIF32"/>
+                <value value="3" name="U8"/>
+                <value value="4" name="U16"/>
+                <value value="5" name="S8"/>
+                <value value="6" name="S16"/>
+            </bitfield>
+            <!-- repeated every 4 bits (8 times for each RT) -->
```

Comment:
If this repeats 8 times, let's make a `<bitfield>` entry for every instance and put the values in a separate enum definition.

Reply:
Changed this and also changed the names to from `S8/16` to `I8/16`